runtime/src/main/cpp/KString.cpp

```diff
@@ -44,7 +44,7 @@ OBJ_GETTER(utf8ToUtf16Impl, const char* rawString, const char* end, uint32_t cha
   if (rawString == nullptr) RETURN_OBJ(nullptr);
   ArrayHeader* result = AllocArrayInstance(theStringTypeInfo, charCount, OBJ_RESULT)->array();
   KChar* rawResult = CharArrayAddressOfElementAt(result, 0);
-  auto convertResult = conversion(rawString, end, rawResult);
+  conversion(rawString, end, rawResult);
```

Comment:
Does the compiler complain about ignoring return value?

Reply:
Unused variable `convertResult`.

Reply:
I get this. I mean, if we remove this variable, is the compiler totally ok with ignoring return value?

Reply:
Well, the function is not marked [`nodiscard`](https://en.cppreference.com/w/cpp/language/attributes/nodiscard) and has a trivial destructor, so it's ok to ignore the return value.

Reply:
I see. Thanks for explanation!